phpcs.xml

```diff
@@ -20,4 +20,14 @@
     <rule ref="LaminasCodingStandard"/>
     <exclude-pattern>test/TestAsset/**</exclude-pattern>
     <exclude-pattern>test/**/TestAsset/**</exclude-pattern>
+
+    <!-- Needed to avoid breaking code for 7.4 since $instance::class only works in PHP 8.0 -->
```

Comment:
```xml
<config name="php_version" value="70400" />
```
This option should solve issue of compat with PHP 7.4

Reply:
I don't get why we would need this in the first place. Coding Standard should only be executed on PHP 7.4, so no need to configure the version at all.

If you can't manage your local environment to run under PHP 7.4, then you might check how to achieve this instead of enforcing config parameters which might be overseen when bumping to PHP 8.0